mw-webapp/src/logic/wayPage/reportsTable/modalContentLabels/ModalContentLabels.module.scss

```diff
@@ -24,27 +25,33 @@ $descriptionMaxWidthMobile:150px;
 
 .labelContent {
   align-items: center;
+  justify-content: space-between;
   gap: $gapMedium;
 }
 
 .labelContainer {
+  display: flex;
```

Comment:
maybe better to update div => HorizontalContainer instead property flex

Reply:
done